docs/source/api/deepinv.datasets.rst

```diff
@@ -0,0 +1,49 @@
+deepinv.datasets
+================
+
+This subpackage can be used for generating reconstruction datasets from other base datasets (e.g. MNIST or CelebA).
```

Comment:
Why do some API sections have a short description and some don't (e.g. Models)?

Reply:
solved

Reply:
now we can use .. userguide:: macros